RocketCastTests/XMLParserTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  XMLParserTests.swift
+//  RocketCast
+//
+//  Created by James Park on 2016-09-11.
+//  Copyright © 2016 UBCLaunchPad. All rights reserved.
+//
+
+import Foundation
+
+import XCTest
+@testable import RocketCast
+
+class XMLParserTests: XCTestCase {
```

Comment:
Should consider making a test for concurrent parsing if it's easy. That is, can we parse 3 XML items at the same time? This is important because we could potentially need to parse multiple podcasts on launch of the device when it refreshes